### PR TITLE
Fix current contribution documentation

### PR DESCRIPTION
**Pull request by dakshshah96**
cc #11168 

- [x] Fix typos
- [x] Fix grammar
- [x] Fix organization

These are just minor changes I've made until I rewrite the whole contribution document.

cc @PeterDaveHello

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,167 +1,113 @@
-﻿# CDNJS contributing document
+﻿# Contribute to CDNJS
 
- - Please read the instructions about sparseCheckout and shallow clone before any git clone operation to this huge repository: [documents/sparseCheckout.md](https://github.com/cdnjs/cdnjs/blob/master/documents/sparseCheckout.md), especially if you are using MacOS which uses case-insensitive filesystem by default, will bring you other issues.
- - Please make sure you are not going to submit a personal library, we have popularity requirement for the libraries, for the detail, see: [issue](#b-request-a-new-library)
+* Read the [sparseCheckout documentation](/documents/sparseCheckout.md) before you clone the CDNJS repo locally. It is not recommended to clone the whole repo as is due to its large size. Using sparse-checkout also works around issues on case-insensitive file systems (HFS+ on macOS).
```
Comment (PeterDaveHello) [inline CONTRIBUTING.md]: Actually, sparseCheckout is about about the clone process, the shallow clone is. sparseCheckout is about not to checkout all the files.

**Comment by PeterDaveHello** [comment]
Dear all members of @cdnjs/team-cdnjs @cdnjs/intern3 @cdnjs/active-contributor @cdnjs/library-reviewer,

Please join us to improve the documentation, even is just a small correction or improvement, thank you all!

**Comment by dakshshah96** [comment]
Just for everyone's information, this is just a PR to fix the current version of the documentation. I haven't added anything new to it. The new documentation which I'll write once this one is fixed will be much more thorough and better organized. Cheers!

**Comment by PeterDaveHello** [comment]
@dakshshah96 maybe I didn't describe my thoughts clear enough, my apologizes, I originally expect a not so large change, just typos, "wrong" grammars, small fixes, without rewrite, reorder, which should be small and easier to be reviewed, and put the big changes in the wiki, what do you think?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,167 +1,113 @@
...
+* [`package.json`](https://www.npmjs.org/doc/package.json.html) file
+    * The root of each library (for example, `ajax/libs/jquery`) must have a the `package.json` file. We store all library-related meta data in this file.
+    * Use information from the official `package.json` file if the library has one.
+    * `package.json` follows 2 space indents.
+    * Use [JSONLint](http://jsonlint.com/) to validate your `package.json` structure.
+* The library's directory name (located at `ajax/libs`) must be same as the `package.json` `name` field.
+* All libraries must have [auto-update](/documents/autoupdate.md) set up in the `package.json` file. This means we don't have to manually add new releases. You'll however need to add files for the latest version manually when adding a new library.
+* Run `npm test` locally and make sure all [tests pass](/README.md#run-npm-test-to-check-all-is-well) (learn how to [install test dependencies](/README.md#install-npm-test-dependencies)).
```
Comment (maruilian11) [inline CONTRIBUTING.md]: Maybe we can remind users to run `npm test` in the root.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,167 +1,113 @@
...
+* Tag related [people](https://github.com/blog/821-mention-somebody-they-re-notified), [issues](https://github.com/blog/957-introducing-issue-mentions) and commits while creating a new issue to make collaboration easier.
 
- * a. [general conventions](#a-general-conventions)
+#### Popularity Requirements for New Libraries
 
- * b. [request a new library](#b-request-a-new-library)
+A library must satisfy the following criteria to be eligible for CDNJS:
+* 100 stars on GitHub **or** 500 downloads in the last month on npm
+* Should not be a personal hobby project
+* Must have at least one official and publicly accessible repository
```
Comment (maruilian11) [inline CONTRIBUTING.md]: @PeterDaveHello Do we request libs must be from `NPM Package` or `GitHub` for the auto-update mechanism?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,167 +1,113 @@
...
+#### General Conventions for Pull Requests
 
- * e. [adding a new library by a single package.json](#e-adding-a-new-library-by-a-single-packagejson)
+1. CDNJS only hosts library with a production/stable release. Exceptions can be made if your pre-production version (alpha/beta/RC) is [popular](#popularity-requirements-for-new-libraries) and you'd like for it to be hosted on CDNJS.
+2. Rebase your CDNJS fork with upstream prior to every time you create a branch for a PR.
+3. Always edit on a branch. Never, ever make edits to the `master` branch. Always make a new branch before you edit files. We follow the [GitHub Flow](https://guides.github.com/introduction/flow/index.html) workflow.
 
- * f. [pre-flight checks](#f-pre-flight-checks)
+4. Don't make multiple unrelated changes in a single commit. Only strongly related changes must be included in a single commit.
+5. Every commit should be meaningful. For example, don't create a new commit to fix a typo you made in your previous commit.
```
Comment (maruilian11) [inline CONTRIBUTING.md]: Maybe we can mention the `rebase` mechanism to let people not be afraid to use a new commit to fix the previous one.

**Comment by PeterDaveHello** [comment]
@maruilian11 we'll update the content in future, this PR should focus on fixing English problem IMO.

File: CONTRIBUTING.md (hunk 4 above)
Comment (mercyg) [inline CONTRIBUTING.md]: "In case of case-insensitive" grammatical change ====> "In the case of case-insensitive"  Thanks

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,167 +1,113 @@
...
+3. Ensure that `filename` in the `package.json` file is correct. File names might change with a new version.
+4. Add [auto-update config](/documents/autoupdate.md) for the library if it doesn't already exist.
 
-* Before opening a issue ticket, please check if there is/was already an issue on the same topic.
+#### Add a New Library With Assets
 
-* If you are going to open a issue about your library (means you are the author/maintainer or one of the main authors/maintainers), please add `[author]` on the issue topic, we'll give it a higher priority to process.
+* All libraries are stores in the `ajax/libs` directory. Each library has it's own directory in `ajax/libs` and each version of the library has it's own subdirectory. For example, jQuery v1.1.2 will be located at `ajax/libs/jquery/1.1.2/`.
+* [`package.json`](https://www.npmjs.org/doc/package.json.html) file
+    * The root of each library (for example, `ajax/libs/jquery`) must have a the `package.json` file. We store all library-related meta data in this file.
```
Comment (mercyg) [inline CONTRIBUTING.md]: "it's own directory .... it's own" ====>  change the it's into possessive  pronoun: "its own .... its own"

Thanks

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,167 +1,113 @@
...
-* Sparse checkout and shallow clone can totally raise efficient of working with cdnjs repo and avoid the problem of case-insensitive filesystem, for detail reason, see it below.
-* The minor reason that you should use sparse checkout and shallow clone:
-  * There is no need to clone the whole history of cdnjs repo if you are not tracing issues or maintaining this project, and there is no need to checkout all the libs on cdnjs for the same reason(unless you want to access the libs without the Internet).
-* And the major reason:
-  * The size of the git repo behind cdnjs is about 2.7GB, but the extracted files will use about 70GB space on your filesystem. If you are using case-insensitive filesystem(usually on Windows and MacOS), you may have problem with [**S**ortable](https://github.com/cdnjs/cdnjs/tree/master/ajax/libs/Sortable) and [**s**ortable](https://github.com/cdnjs/cdnjs/tree/master/ajax/libs/sortable), we have an issue disscussion here: [#3650](https://github.com/cdnjs/cdnjs/issues/3650)
+* Have you complied with all our conventions?
+* Have you followed the library's directory structure?
+* Does a valid and accurate `package.json` file exist for the library?
```
Comment (mercyg) [inline CONTRIBUTING.md]: "file exist for the library"  grammar change ====> file exists...
Thanks

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,167 +1,113 @@
...
+* Have you minified CSS and JavaScript assets?
+* Did all tests pass when you ran `npm test`?
 
+#### Miscellaneous Information
 
+* We recommend working on a Unix-like environment (for example, GNU/Linux or BSD distributions except macOS) due to how git works.
```
Comment (mercyg) [inline CONTRIBUTING.md]: "working on" grammatical change "working in "  
"except macOS" grammer==== "except for macOS"
Thanks

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,167 +1,113 @@
...
+* If this document didn't answer a question of yours, refer to [CONTRIBUTING-WIP.md](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING-WIP.md) (might be outdated). In case of any conflicts, follow CONTRIBUTING.md. Also, open an issue and report this if you can.
```
Comment (mercyg) [inline CONTRIBUTING.md]: "In case of any conflicts" grammar  ==== > "In the case of any conflicts"

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -1,167 +1,113 @@
...
+* New contribution documentation is in the works and will be released soon.
+* Add auto-update configuration for all libraries if it doesn't exist. This is irrespective of whether you are updating or adding the library.
+* Pro tip: Use `git diff` before creating to commit to see the changes you have made. You can also use `git log -p` or `git show sha1hash` to differentiate.
```
Comment (mercyg) [inline CONTRIBUTING.md]: "creating to commit to see the change"  grammar change into:-  "creating commit to see the change"

**Comment by sufuf3** [comment]
@dakshshah96 Would you fix some grammar?
Thank you.

**Comment by PeterDaveHello** [comment]
@sufuf3 I doubt if you understand the whole discussion? cc @extend1994

**Comment by maruilian11** [comment]
Agree with Meheret.

**Pull request closed without merging**